frontend/packages/dev-console/src/components/topology/filters/filter-utils.ts

```diff
@@ -18,11 +19,10 @@ export enum ExpandFiltersKeyValue {
   operatorGrouping = 'Operator Groupings',
 }
 
-export const getTopologyFilters = ({
-  plugins: {
-    devconsole: { topology },
-  },
-}: RootState): TopologyFilters => topology.get('filters');
+export const getTopologyFilters = (state: RootState): TopologyFilters => {
+  const topology = state?.plugins?.devconsole?.topology;
+  return topology ? topology.get('filters') : DEFAULT_TOPOLOGY_FILTERS;
```

Comment:
@jeff-phillips-18 If redux store is not initialized and `topology` is undefined. Shouldn't we check local storage for filters because filter values might be different from DEFAULT_TOPOLOGY_FILTERS? WDYT?

Reply:
Agreed. I will update.